src/libecalc/core/consumers/legacy_consumer/system/results.py

```diff
@@ -30,7 +31,7 @@ def energy_usage(self) -> List[Optional[float]]:
         return self.consumer_model_result.energy_usage
 
     @property
-    def power(self) -> NDArray[np.float64]:
+    def power(self) -> PydanticNDArray:
```

Comment:
You suggest to remove np.float64 as annotation? I guess it might have been too much...and not really useful?

Reply:
No, I think np.float64 notation should stand:-)

Reply:
I see. I guess this is the same as has been done other places, so should be ok here as well. We should rather spend time fixing it properly as we discussed earlier, just get this fix in place now (wo. float64 then) :)

Reply:
Super, agree. I checked with PydanticNDArray[np.float64], but it failed (BeforeValidator I think).